### PR TITLE
Load program even if the user is not an operator

### PR DESCRIPTION
**Pull request by boudra**
## Description

I added the roles filter in the GraphQL but this makes it difficult to debug or see programs that are not yours through the debug mode, if we load the program but then check the roles in the code we can implement this easier.

## Checklist

This PR:

- [x] Includes only changes relevant to the original ticket. Significant refactoring needs to be separated.
- [x] Doesn't disable eslint rules.
- [x] Doesn't work around the type checker (including but not limited to: type casts, non-null assertions, `@ts-ignore`, unjustified optional values).
- [x] Doesn't contain commented out code.
- [x] If adding/updating a feature, it adds/updates its test script on Notion.

### CODE REVIEW

File: packages/data-layer/src/queries.ts [hunk 1]
```diff
@@ -312,17 +307,12 @@ export const getRoundByIdAndChainId = gql`
   }
 `;
 
-export const getRoundsByProgramIdAndUserAddress = gql`
-  query getRoundsByProgramIdAndMemberAddress(
-    $chainId: Int!
-    $programId: String!
-    $userAddress: String!
-  ) {
+export const getRoundsByProgramId = gql`
```
Comment (thelostone-mc) [inline packages/data-layer/src/queries.ts:310]:
```suggestion
export const getRoundsByProgramIdAndChainId = gql`
```
If we are adding a chainId filter, would recommend updating this to getRoundsByProgramIdAndChainId as in v2 we can have rounds across chains by the same program/profile

Comment (thelostone-mc) [reply packages/data-layer/src/queries.ts:310]: Would recommend doing the same for the above 2 queries : getProgramByIdAndUser, getProgramByUserAndTag to make it easier to read

Comment (boudra) [reply packages/data-layer/src/queries.ts:310]: I think it's a good point but this PR is not adding a chain ID filter, it's just removing the roles filter, renaming other queries is out of scope of this PR, we can do it in a separate one

Comment (boudra) [reply packages/data-layer/src/queries.ts]: I updated the one I changed though since I am already renaming it, thanks for the suggestion!

**Review by vacekj — APPROVED** [review]
:shipit:

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**